e2e/content-services/upload/upload-dialog.e2e.ts

```diff
@@ -105,8 +105,8 @@ describe('Upload component', () => {
     });
 
     afterEach(async () => {
-        const nbResults = await contentServicesPage.numberOfResultsDisplayed();
-        if (nbResults > 1) {
+        const nbResults = await contentServicesPage.emptyFolder.isPresent();
+        if (nbResults) {
```

Comment:
initially, if the numberOfResultsDisplayed was bigger than 1, the body of the IF was executed, but now, the body of the IF is executed when emptyFolder.isPresent .... 
should this be `if (!nbResults)` ?